<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VignetteRecette</class>
 <widget class="QWidget" name="VignetteRecette">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>330</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#VignetteRecette
{
	border: 3px solid black;
	background-color :lightgreen;
	border-radius:20px;
}
QLabel
{
	font: bold 18px;
}

QWidget#VignetteRecette:hover
{
	border: 3px solid black;
	background-color :blue;
	border-radius:20px;
}           </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="buttonSuppr">
     <property name="text">
      <string>Supprimer recette</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelNom">
     <property name="text">
      <string>Gateau au chocolat</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelImage">
     <property name="styleSheet">
      <string notr="true">QLabel#labelImage

{
	border-image: url(:/Images/Images/gateauChocolat.jpg) 0 0 0 0 stretch stretch;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Durée de préparation :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelDureePrep">
       <property name="text">
        <string>30 min</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Réalisable :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelRealisable">
       <property name="text">
        <string>oui</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
